src/templates/chat/roll-result.html

```diff
@@ -8,13 +9,28 @@
             <div class="chat-img" style="background-image:url('{{data.actor.img}}')"></div>
             {{/if}}
         </div>
+        <div class="card-content">
+            {{{data.description}}}
+        </div>
         <div class="blindable" data-blind="{{data.roll.blindroll}}">
             {{#if result.details}}<div class="chat-details">{{{result.details}}}</div>{{/if}}
             {{#if result.isFailure}}<div class='roll-result roll-fail'><b>{{localize 'OSE.Failure'}}</b> ({{result.target}})
             </div>{{/if}}
             {{#if result.isSuccess}}<div class='roll-result roll-success'><b>{{localize 'OSE.Success'}}</b>
                 ({{result.target}})</div>{{/if}}
             {{#if rollOSE}}<div>{{{rollOSE}}}</div>{{/if}}
+            <div class="card-buttons">
+                {{#if data.save}}
```

Comment:
Thoughts on moving this `#if` up to surround `.card-buttons`?  Unless you expect to see more buttons in this card.

Reply:
Since no buttons should exist but saves I think this is reasonable. Resolved in https://github.com/vttred/ose/pull/497/commits/711998f72d185aae1fe4a40074c3efcc2b45f220